### PR TITLE
use only y2storage classes and methods

### PR DESCRIPTION
**Pull request by wfeldt**

### CODE REVIEW

**Comment by ancorgs** [comment]
LGTM. Thanks for adding the comment

File: src/lib/installation/prep_shrink.rb [hunk 1]
```diff
@@ -48,6 +56,9 @@ def run(*args)
 
     MAXIMAL_SIZE_KB = 8192
     def shrink_partitions
+# storage-ng
+# rubocop:disable Style/BlockComments
+=begin
```
Comment (ancorgs) [inline src/lib/installation/prep_shrink.rb]: Is this already documented in installer_hacks?

File: src/lib/installation/prep_shrink.rb [hunk 2]
```diff
@@ -58,6 +69,7 @@ def shrink_partitions
           Yast::SCR.Execute(YAST_BASH_PATH, cmd)
         end
       end
+=end
```
Comment (ancorgs) [inline src/lib/installation/prep_shrink.rb]: Nitpicking: we should probably re-enable rubocop

File: src/lib/installation/prep_shrink.rb (hunk 1 above)
Comment (wfeldt) [inline src/lib/installation/prep_shrink.rb]: https://github.com/yast/yast-storage-ng/pull/267

File: src/lib/installation/prep_shrink.rb (hunk 2 above)
Comment (wfeldt) [inline src/lib/installation/prep_shrink.rb]: It will be gone anyway when the block is removed.

**Comment by ancorgs** [comment]
LGTM again

**Pull request merged**